applications/tari_console_wallet/README.md

```diff
@@ -263,3 +263,5 @@ Run a series of commands from a given script. The commands should be formatted t
 ## Recovery mode
 
 todo docs
+
+#
```

Comment:
?

Reply:
I will remove it. It was a test for my first verified commit.